crates/core/src/format.rs

```diff
@@ -160,7 +160,7 @@ impl Format {
             RealizedHref::Url(url) => {
                 use object_store::ObjectStore;
 
-                let (object_store, path) = object_store::parse_url_opts(&url, options)?;
+                let (object_store, path) = parse_url_opts(&url, options)?;
```

Comment:
It would be nice to have some rust API that allows the user to pass in their own `Arc<dyn ObjectStore>` instead of constructing the store from inside your code

Reply:
https://github.com/stac-utils/stac-rs/issues/643